poetry.lock

```diff
@@ -3113,7 +3092,7 @@ python-versions = "*"
 
 [[package]]
 name = "tensorflow"
-version = "2.11.0"
+version = "2.11.1"
```

Comment:
@varunshankar Judging from the CI failures for Python 3.7 on both Ubuntu and Windows, it appears that Python 3.7 support was removed from this TF micro (although their [release changelog](https://github.com/tensorflow/tensorflow/releases/tag/v2.12.0) only mentions it in the recent `2.12.0`release 😕 ) I'd advise pinning Tensorflow to `2.11.0`. If you could also submit a ticket to TF that would be great so we make them aware of this.

Reply:
Thanks a lot for the help @ancalita. If I pin the tensorflow version, the `poetry update` command is stuck in resolving dependencies. It has run for an hour and its still resolving dependencies. I tried running it 2-3 times and I still face this issue.

Reply:
Could you check the poetry version you're using? The stuck step of resolving dependencies usually happens if you've exited some poetry command by force. You can use `poetry cache clear` [command](https://python-poetry.org/docs/cli/#cache-clear). In addition to `pypi`, I also clear `_default_cache`, but you can have a look what caches you have with `poetry cache list`. Hope it helps!

Reply:
After the cache clear, you can try `poetry update tensorflow` again.